Test/SmaVariable.Test.ps1

```diff
@@ -0,0 +1,69 @@
+﻿$Module = "$PSScriptRoot\..\DSCResources\xSmaVariable\xSmaVariable.psm1"
+
```

Comment:
Suggestion: Base the tests on the [Unit tests Template](https://github.com/PowerShell/DscResources/blob/master/Tests.Template/unit_template.ps1